inst/prepare_package_cran.sh

```diff
@@ -0,0 +1,24 @@
+echo "==> Switching to cran-release branch."
+cd ..
```

Comment:
First make sure to go into script directory:
`BASEDIR=$(dirname "$0")`

Your current code don't work if run from other directory (e.g. from / run `$>./inst/prepare_package_cran.sh`)